index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <title aria-label="title">Graphic Designer's Summit Website</title>
+  </head>
+  <body>
+    <!-- Navigation Section  -->
+
+    <header>
+      <nav class="top-header">
+        <ul class="top-nav">
+          <li class="top-item">
+            <a href="#" class="nav-link"
+              ><img src="img/facebook-icon.png" alt=""
+            /></a>
+          </li>
+          <li class="top-item">
+            <a href="#" class="nav-link"><img src="img/t-icon.png" alt=""/></a>
+          </li>
+          <li class="top-item">
+            <a href="#" class="nav-link">English</a>
+          </li>
+          <li class="top-item">
+            <a href="#" class="nav-link">login</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav class="navbar">
+        <a href="index.html" class="nav-branding"
+          ><img src="img/logo1.png" alt="logo image"
+        /></a>
+        <ul class="nav-menu nav-menu1">
+          <li class="nav-item home-href">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+            <a href="about.html" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact-me" class="nav-link">Join</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Sponsor</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">News</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact-me" class="nav-link">
+              <span> GD Campaign</span></a
+            >
+          </li>
+        </ul>
+
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
```

Comment:
### Your hamburger Icon
![image](https://user-images.githubusercontent.com/93039359/210936201-6ba819a7-0be0-4338-8df2-797994996935.png)

### Design hamburger Icon
![image](https://user-images.githubusercontent.com/93039359/210936633-2fb854c1-e3e1-4ff8-8795-4743267c2a6b.png)


- [ ] You hamburger icon has a background and then you homepage has a different background which is not according to the design. If you compare your project with cindy shin design then you would see that there is only one background which is covering the hamburger icon and homepage as well :+1: I have given some instructions below how you can make it :blush:
- [ ] Kindly make these changes for hambuger icon for both index.html and about.html files :blush:

Step 1
![image](https://user-images.githubusercontent.com/93039359/210936070-1b802d59-3de3-4656-8ef9-d05baa9bfb6a.png)

Step 2
![image](https://user-images.githubusercontent.com/93039359/210936137-0c611fe3-f5d3-4f3d-aa45-84a2bcd8816f.png)

### After making those changes
![image](https://user-images.githubusercontent.com/93039359/210936362-92af0ef2-0d95-49f3-ba9e-9fdedf0f2ca7.png)

Reply:
I have implemented the required changes